### PR TITLE
Writing style.

### PR DESCRIPTION
**Pull request by ezra-at-lumedic**
While I'm not a documentation writer by profession, I am something of a wordsmith and the current state of the documentation makes me cringe.  
I understand if English is not a first language this can be challenging, and am happy to add my 2cs where it might help.  
The intro has run-on sentences, bad grammar, lack of clarity, etc., etc. and so I am submitting this.  
BUT.... please, consider enlisting someone who is a doc author to help.  These docs are currently really rough given the relative maturity of the software.

### CODE REVIEW

File: overview/life-of-a-fluentd-event.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ complete cycle including *Setup*, *Inputs*, *Filters*, *Matches* and
 
 ## Basic Setup
 
-The configuration file is the fundamental piece to connect all things
-together, as it allows to define which *Inputs* or listeners
-[Fluentd](http://fluentd.org) will have and setup common matching rules
-to route the *Event* data to a specific *Output*.
+The configuration file contains the information that connect together all the components.
```
Comment (ganmacs) [inline overview/life-of-a-fluentd-event.md]: [nits] trailing space

File: overview/life-of-a-fluentd-event.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ complete cycle including *Setup*, *Inputs*, *Filters*, *Matches* and
 
 ## Basic Setup
 
-The configuration file is the fundamental piece to connect all things
-together, as it allows to define which *Inputs* or listeners
-[Fluentd](http://fluentd.org) will have and setup common matching rules
-to route the *Event* data to a specific *Output*.
+The configuration file contains the information that connect together all the components. 
+It contains the collection of *Inputs* or listeners
```
Comment (ganmacs) [inline overview/life-of-a-fluentd-event.md]: [nits] `*Input*s` ?

**Comment by ganmacs** [comment]
Can you please follow the DCO? https://github.com/fluent/fluentd-docs-gitbook/pull/164/checks?check_run_id=534579284

**Comment by ezra-at-lumedic** [comment]
Not worth my time

Sent from my mobile, please expect typos.
________________________________

**Pull request closed without merging**